components/form/Input.js

```diff
@@ -20,23 +20,34 @@ const Input = forwardRef(
       }
     };
 
```

Comment:
first we need to declare a variable
```suggestion
let inputClassName = ""
```